s3/src/test/resources/Wasabi (us-central-1).cyberduckprofile

```diff
@@ -46,6 +46,7 @@
         <key>Properties</key>
         <array>
             <string>s3.storage.class.options=STANDARD</string>
+            <string>s3.storage.class.options=STANDARD</string>
```

Comment:
Delete.

Reply:
Added to make sure test case fails without fix.